account_reconciliation_widget/models/reconciliation_widget.py

```diff
@@ -340,18 +334,17 @@ def get_bank_statement_data(self, bank_statement_line_ids, srch_domain=None):
             [line["st_line"]["id"] for line in results["lines"]]
         )
         bank_statements_left = bank_statement_lines_left.mapped("statement_id")
-
+        data = bank_statements_left.read(["name", "journal_id"])
+        data = data and data[0]
         results.update(
             {
                 "statement_id": len(bank_statements_left) == 1
-                and bank_statements_left.id
+                and bank_statements_left[0].id
```

Comment:
Maybe better to use `fields.first`. If the recordset is empty this would fail.

Reply:
It's not going to fail, as there's a check before, and `.id` doesn't need to be fetched from DB, so it's OK this way.